docker/jenkins-images/Dockerfile-graalvm

```diff
@@ -1,4 +1,4 @@
-# This is not part of the automated system (Will be fixed by https://h2oai.atlassian.net/browse/ITA-1506)
```

Comment:
ITA are not converted?

Reply:
No, they are unimportant - just technical or devops tasks.